karma-min.conf.js

```diff
@@ -33,6 +33,7 @@ module.exports = function(config) {
       'bower_components/angular-xeditable/dist/js/xeditable.js',
       'bower_components/angular-toastr/dist/angular-toastr.tpls.js',
       'bower_components/bootstrap/dist/js/bootstrap.js',
+      'bower_components/ng-tags-input/ng-tags-input.js',
```

Comment:
Yeah if it is a bower_component then it should be committed. But as I cannot see it in the bower.json I thought it was a mistake. I just tested though and it's writing it to my karma files on gulp test too.. Must be a sub-dep